bitnami/rabbitmq/templates/NOTES.txt

```diff
@@ -50,7 +51,7 @@ To Access the RabbitMQ AMQP port:
 1. Obtain the NodePort IP and ports:
 
     export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
-    export NODE_PORT_AMQP=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[1].nodePort}" services {{ include "rabbitmq.fullname" . }})
+    export NODE_PORT_AMQP=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "rabbitmq.fullname" . }})
```

Comment:
To avoid more issues with the index, have you  try something like (not sure if it will work):

```{.spec.ports[?(@.name== 'amqp')].nodePort}```

Reply:
It worked like a charm! Thanks for the suggestion